docs/_templates/indexcontent.html

```diff
@@ -0,0 +1,47 @@
+{% extends "defindex.html" %}
+
+{% block body %}
+
+<div class="figure align-center" id="id1">
+    <img src="_images/logo.svg">
+    <h1>Kinto — Store, Sync, Share and Self-Host.</h1>
+    <p>Kinto is a lightweight JSON storage service, with synchronisation and sharing abilities</p>
+</div>
+
+<div id="toc">
+    <div class="item">
+          <h3><a href="{{ pathto("overview") }}">Overview</a></h3>
+          <p class="description">Get a high level overview of what Kinto is.</p>
```

Comment:
*Overview* is used twice here. Consider something like:
> A high-level look at Kinto.
An introduction to the Kinto universe.
A high-level introduction.

etc.